CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,77 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to make participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+  advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any contributor for other behaviors that they deem inappropriate,
+threatening, offensive, or harmful.
+
+## Scope
+
+This Code of Conduct applies within all project spaces, and it also applies when
+an individual is representing the project or its community in public spaces.
+Examples of representing a project or community include using an official
+project e-mail address, posting via an official social media account, or acting
+as an appointed representative at an online or offline event. Representation of
+a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [psc@proj.org](mailto:psc@proj.org). All
```

Comment:
do we have such a psc@proj.org ? I'd suggest to forward to the OSGeo CodeOfConduct email : coc@osgeo.org

Reply:
No, we don't. This was actually a thing I wanted to discuss. I needed a placeholder. I agree that referring such matters directly to OSGeo officers but I found it unclear from https://www.osgeo.org/code_of_conduct/ whether that is the intended use of that mail address or not.

Reply:
maybe ask @jodygarnett ? I don't think the OSGeo CoC committee is overloaded (from what I remember from the last AGM, they didn't have to address a single cast last year, which is a good thing), so they can potentially take additional 0-activity from OSGeo projects :-)

Reply:
The code of conduct committee needs more volunteers, indeed Michael Smith and myself stepped up last year to help out.  You should not hear about committee activities often. As OSGeo in part promotes professional conduct we try and address any issues via employer / organization which is in a position to enact descrete direct formal censure.

For OSGeo committees (which after graduation PROJ will be recognized as such) the project officer is the key point of contact, covered by osgeo liability insurance etc...

For GeoServer [we took advantage of this](https://github.com/geoserver/geoserver/blob/master/CODE_OF_CONDUCT.md) to make our project office the point of contact for any code-of-conduct reporting.

Reply:
The coc@osgeo.org email address is terrible - it is an unarchived mailing list - so anyone sending a message gets a warning about their message being moderated. I have an open trac issue to get it changed to an alias :P

Reply:
>  For OSGeo committees (which after graduation PROJ will be recognized as such) the project officer is the key point of contact, covered by osgeo liability insurance etc...

Right, I'll put my address in then as I suspect I'll be the project officer once PROJ has graduated. Thanks for the clarification.